homework/content.js

```diff
@@ -1,21 +1,26 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
+(function(exports) {
+  var ContentManager = function(){
+	this._wrapper = null;
+};
 
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
-  }
+ContentManager.prototype = {
 
-  function resetWrapper() {
-    _wrapper.innerHTML = '';
+  start() {
+	  this._wrapper = document.querySelector('#note-content-wrapper');
+      window.addEventListener('note-open', (function(event){
```

Comment:
Instead of using an anonymous function, setting the listener as `this` and handle events with `handleEvent` method is better.